code/game/gamemodes/gang/gang_things.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/gang_induction_package
+	name = "family signup package"
+	icon = 'icons/obj/gang/signup_points.dmi'
+	icon_state = "signup_book"
+	var/gang_to_use
+	var/datum/team/gang/team_to_use
+
+
+/obj/item/gang_induction_package/attack_self(mob/living/user)
+	..()
+	if(HAS_TRAIT(user, TRAIT_MINDSHIELD))
```

Comment:
What if you were implanted partway through a round, though?

Like, you're the HoP, and you went to sec to get mindshielded at shiftstart before the gang announcement. Now you suddenly retroactively have attended a seminar?

What if you get mindshielded after becoming a gangster, then try to switch teams? Can gangs use mindshields to ensure the loyalty of their gang members? That could become an intended/emergent feature if you change the text here.

Reply:
The idea is the mindshield implant totally puts those false memories into your head about gang PSAs.

Reply:
also, you should probably add a check to make sure that space cops and the undercover space cop can't join gangs

Reply:
> The idea is the mindshield implant totally puts those false memories into your head about gang PSAs.

eh, I'm not sure if that fits our current mindshield implant   L O R E

they're less "brainwashing implants" and more "implants that protect important people from being brainwashed like in that part of the Allied campaign of Red Alert 2", IMO

Reply:
I think goof is stuck in the era when mindshield implants were loyalty implants and the lore was that they brainwashed you into being extremely loyal to NT. It got changed because it was unclear with antagonists because "i loyal to NT i no longer know about my syndicate employment".